contrib/download-frozen-image-v2.sh

```diff
@@ -201,6 +201,58 @@ handle_single_manifest_v2() {
 	manifestJsonEntries=("${manifestJsonEntries[@]}" "$manifestJsonEntry")
 }
 
+get_target_arch() {
+	if [ -n "${TARGETARCH}" ]; then
+		echo "${TARGETARCH}"
+		return 0
+	fi
+
+	if type go > /dev/null; then
+		echo "$(go env GOARCH)"
+		return 0
+	fi
+
+	if type dpkg > /dev/null; then
+		debArch="$(dpkg --print-architecture)"
+		case "${debArch}" in
+			armel|armhf)
+				echo "arm"
+				return 0
+				;;
+			*)
+				echo "${debArch}"
+				return 0
+				;;
+		esac
+	fi
+
+	if type uname > /dev/null; then
+		uArch="$(uname -m)"
+		case "${uArch}" in
+			x86_64)
+				echo amd64
+				return 0
+				;;
+			arm|armv6l|armv7l)
```

Comment:
looks like shfmt is configured to want spaces here. Either update this script or we need to look what option to set on shftm to format without spaces (either way would be fine with me)

![9EC68492-A6A4-42EC-8749-AAF4F02DED14](https://user-images.githubusercontent.com/1804568/93662868-a24ec900-fa63-11ea-8b69-88bc13241882.jpeg)